src/string_util.cc

```diff
@@ -155,6 +140,22 @@ std::string StrFormatImp(const char* msg, va_list args) {
   return std::string(buff_ptr.get());
 }
 
+}  // end namespace
+
+void AppendHumanReadable(int n, std::string* str) {
+  std::stringstream ss;
+  // Round down to the nearest SI prefix.
+  ss << ToBinaryStringFullySpecified(n, 0);
```

Comment:
Just a question. Why are we not sending `Counter::kIs1000?` Or will we make this change afterwards?

Reply:
i wanted to get the barest minimal fixes for what's currently in the code before changing behaviour (aside from bug fixes).

Reply:
Got it.